lib/proxy/oauth/signatures/oauth1a.js

```diff
@@ -40,9 +40,9 @@ function normaliseRequestParams(argument_pairs) {
       continue;
     }
     **/
-    args += sprintf("%s=%s", argument_pairs[i][0], argument_pairs[i][1]);
+    args += sprintf("%s%%3D%s", argument_pairs[i][0], argument_pairs[i][1]);
```

Comment:
Never mind, RFC is pretty clear on this point: http://oauth.net/core/1.0a/#anchor13